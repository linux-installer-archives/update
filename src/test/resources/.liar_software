#!/bin/bash

#
# Copyright (C) 2017 Pierre-Yves Fourmond.

# This file is part of liar, the Linux Installer for ARchives.

# liar is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.

# liar is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU General Public License for more details.

# You should have received a copy of the GNU General Public License
# along with liar. If not, see <http://www.gnu.org/licenses/>.
#

#
# This file is used by liar-completion, which defines Bash tab auto-completion rules for the liar command.
#

LIAR=.liar_software
VERSION=0.2-dev

if [ "$1" = "version" ]; then
    echo "$LIAR version $VERSION"
fi

software_list=(
ant
atom
clion
evans
firefox
go
goland
graalvm
gradle
groovy
grpcurl
hugo
icecat
idea_community
idea_edu
idea_ultimate
jdk9
jdk10
jdk11
jdk12
jdk13
jdk14
jdk_ea15
jdk_ri7
jdk_ri8
jdk_ri9
jdk_ri10
jdk_ri11
jdk_ri12
jdk_ri13
jdk_ri14
marp_cli
maven
node
postman
pycharm_community
pycharm_edu
pycharm_professional
robo3t
rubymine
sbt
scala
spring
sublimemerge
sublimetext
torbrowser
visualvm
webstorm
xsv
)

ant=https://www-eu.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.zip
atom=https://github.com/atom/atom/releases/download/v1.35.1/atom-amd64.tar.gz
clion=https://download.jetbrains.com/cpp/CLion-2020.1.0.tar.gz
evans=https://github.com/ktr0731/evans/releases/download/0.8.4/evans_linux_amd64.tar.gz
firefox=https://download-installer.cdn.mozilla.net/pub/firefox/releases/66.0.2/linux-x86_64/fr/firefox-66.0.2.tar.bz2
go=https://dl.google.com/go/go1.12.2.linux-amd64.tar.gz
goland=https://download.jetbrains.com/go/goland-2018.3.6.tar.gz
graalvm=https://github.com/oracle/graal/releases/download/vm-19.2.1/graalvm-ce-linux-amd64-19.2.1.tar.gz
gradle=https://downloads.gradle.org/distributions/gradle-5.3.1-bin.zip
groovy=https://dl.bintray.com/groovy/maven/apache-groovy-binary-2.5.6.zip
grpcurl=https://github.com/fullstorydev/grpcurl/releases/download/v1.4.0/grpcurl_1.4.0_linux_x86_64.tar.gz
hugo=https://github.com/gohugoio/hugo/releases/download/v0.54.0/hugo_0.54.0_Linux-64bit.tar.gz
icecat=https://ftp.gnu.org/gnu/gnuzilla/60.3.0/icecat-60.3.0.en-US.gnulinux-x86_64.tar.bz2
idea_community=https://download.jetbrains.com/idea/ideaIC-2018.3.6.tar.gz
idea_edu=https://download.jetbrains.com/idea/ideaIE-2020.0.tar.gz
idea_ultimate=https://download.jetbrains.com/idea/ideaIU-2018.3.6.tar.gz
jdk10=https://download.java.net/java/GA/jdk10/10.0.2/19aef61b38124481863b1413dce1855f/13/openjdk-10.0.2_linux-x64_bin.tar.gz
jdk11=https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz
jdk12=https://download.java.net/java/GA/jdk12/GPL/openjdk-12_linux-x64_bin.tar.gz
jdk_ri7=https://download.java.net/openjdk/jdk7u75/ri/openjdk-7u75-b13-linux-x64-18_dec_2014.tar.gz
jdk_ri8=https://download.java.net/openjdk/jdk8u40/ri/openjdk-8u40-b25-linux-x64-10_feb_2015.tar.gz
jdk_ri9=https://download.java.net/openjdk/jdk9/ri/openjdk-9+181_linux-x64_ri.zip
jdk_ri10=https://download.java.net/openjdk/jdk10/ri/openjdk-10+44_linux-x64_bin_ri.tar.gz
jdk_ri11=https://download.java.net/openjdk/jdk11/ri/openjdk-11+28_linux-x64_bin.tar.gz
marp_cli=https://github.com/marp-team/marp-cli/releases/download/v0.14.1/marp-cli-v0.14.1-linux.tar.gz
maven=http://apache.mediamirrors.org/maven/maven-3/3.6.0/binaries/apache-maven-3.6.0-bin.tar.gz
pycharm_community=https://download.jetbrains.com/python/pycharm-community-2019.1.1.tar.gz
pycharm_edu=https://download.jetbrains.com/python/pycharm-edu-2020.0.tar.gz
pycharm_professional=https://download.jetbrains.com/python/pycharm-professional-2019.1.1.tar.gz
robo3t=https://download.robomongo.org/1.2.1/linux/robo3t-1.2.1-linux-x86_64-3e50a65.tar.gz
rubymine=https://download.jetbrains.com/ruby/RubyMine-2018.3.5.tar.gz
scala=https://downloads.lightbend.com/scala/2.12.8/scala-2.12.8.tgz
sublimetext=https://download.sublimetext.com/sublime_text_3_build_3200_x64.tar.bz2
torbrowser=https://dist.torproject.org/torbrowser/8.0.8/tor-browser-linux64-8.0.8_fr.tar.xz
visualvm=https://github.com/visualvm/visualvm.src/releases/download/1.4.2/visualvm_142.zip
webstorm=https://download.jetbrains.com/webstorm/WebStorm-2018.3.6.tar.gz
xsv=https://github.com/BurntSushi/xsv/releases/download/0.13.0/xsv-0.13.0-x86_64-unknown-linux-musl.tar.gz
