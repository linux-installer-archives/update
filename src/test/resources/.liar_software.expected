#!/bin/bash

#
# Copyright (C) 2017 Pierre-Yves Fourmond.

# This file is part of liar, the Linux Installer for ARchives.

# liar is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.

# liar is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU General Public License for more details.

# You should have received a copy of the GNU General Public License
# along with liar. If not, see <http://www.gnu.org/licenses/>.
#

#
# This file is used by liar-completion, which defines Bash tab auto-completion rules for the liar command.
#

LIAR=.liar_software
VERSION=0.2-dev

if [ "$1" = "version" ]; then
    echo "$LIAR version $VERSION"
fi

software="\
ant atom \
firefox \
go goland gradle groovy \
hugo \
icecat idea_community idea_ultimate \
jdk10 jdk11 jdk12 jdk_ri7 jdk_ri8 jdk_ri9 jdk_ri10 jdk_ri11 \
marp maven \
pycharm_community pycharm_professional \
robo3t rubymine \
scala sublimetext \
torbrowser \
visualvm \
webstorm \
"

ant=http://mirrors.standaloneinstaller.com/apache/ant/binaries/apache-ant-1.10.6-bin.zip
atom=https://github.com/atom/atom/releases/download/v1.35.2/atom-amd64.tar.gz
firefox=https://download-installer.cdn.mozilla.net/pub/firefox/releases/66.0.3/linux-x86_64/fr/firefox-66.0.3.tar.bz2
go=https://dl.google.com/go/go1.12.3.linux-amd64.tar.gz
goland=https://download.jetbrains.com/go/goland-2018.3.7.tar.gz
gradle=https://downloads.gradle.org/distributions/gradle-5.3.2-bin.zip
groovy=https://dl.bintray.com/groovy/maven/apache-groovy-binary-2.5.7.zip
hugo=https://github.com/gohugoio/hugo/releases/download/v0.54.1/hugo_0.54.1_Linux-64bit.tar.gz
icecat=https://ftp.gnu.org/gnu/gnuzilla/60.3.1/icecat-60.3.1.en-US.gnulinux-x86_64.tar.bz2
idea_community=https://download.jetbrains.com/idea/ideaIC-2018.3.7.tar.gz
idea_ultimate=https://download.jetbrains.com/idea/ideaIU-2018.3.7.tar.gz
jdk10=https://download.java.net/java/GA/jdk10/10.0.2/19aef61b38124481863b1413dce1855f/13/openjdk-10.0.2_linux-x64_bin.tar.gz
jdk11=https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz
jdk12=https://download.java.net/java/GA/jdk12/GPL/openjdk-12_linux-x64_bin.tar.gz
jdk_ri7=https://download.java.net/openjdk/jdk7u75/ri/openjdk-7u75-b13-linux-x64-18_dec_2014.tar.gz
jdk_ri8=https://download.java.net/openjdk/jdk8u40/ri/openjdk-8u40-b25-linux-x64-10_feb_2015.tar.gz
jdk_ri9=https://download.java.net/openjdk/jdk9/ri/openjdk-9+181_linux-x64_ri.zip
jdk_ri10=https://download.java.net/openjdk/jdk10/ri/openjdk-10+44_linux-x64_bin_ri.tar.gz
jdk_ri11=https://download.java.net/openjdk/jdk11/ri/openjdk-11+28_linux-x64_bin.tar.gz
marp=https://github.com/yhatt/marp/releases/download/v0.0.15/0.0.15-Marp-linux-x64.tar.gz
maven=http://apache.mediamirrors.org/maven/maven-3/3.6.0/binaries/apache-maven-3.6.0-bin.tar.gz
pycharm_community=https://download.jetbrains.com/python/pycharm-community-2019.1.2.tar.gz
pycharm_professional=https://download.jetbrains.com/python/pycharm-professional-2019.1.2.tar.gz
robo3t=https://download.robomongo.org/1.2.1/linux/robo3t-1.2.1-linux-x86_64-3e50a65.tar.gz
rubymine=https://download.jetbrains.com/ruby/RubyMine-2018.3.6.tar.gz
scala=https://downloads.lightbend.com/scala/2.12.9/scala-2.12.9.tgz
sublimetext=https://download.sublimetext.com/sublime_text_3_build_3200_x64.tar.bz2
torbrowser=https://dist.torproject.org/torbrowser/8.0.9/tor-browser-linux64-8.0.9_fr.tar.xz
visualvm=https://github.com/visualvm/visualvm.src/releases/download/1.4.3/visualvm_142.zip
webstorm=https://download.jetbrains.com/webstorm/WebStorm-2018.3.7.tar.gz
